101-sql-with-failover-group/README.md

```diff
@@ -0,0 +1,11 @@
+# Create a SQL Server with Failover Group on Azure
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-sql-with-failover-group%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-sql-with-failover-group%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template creates two Azure SQL servers, a SQL database, and failover group. Template was authored by John Downs.
```

Comment:
remove the authored by comments since it pulled from metadata in various experiences